src/components/common/Heading/index.js

```diff
@@ -24,7 +33,12 @@ Heading.propTypes = {
   /**
    * Text above the title. It'll be centered inside the title.
    */
-  subtitle: PropTypes.string.isRequired
+  subtitle: PropTypes.string.isRequired,
+
+  /**
+   * Space between characters in Title
+   */
+  letterSpace: PropTypes.number
```

Comment:
This adds buggy functionality. Remove this new feature and set the width of the component from the parent.